conf/application.conf

```diff
@@ -107,8 +104,12 @@ feature-switch {
     released-in-production = false
   }
 
+  postCessationReceipts.enabled = true
```

Comment:
Have added as there is code currently for this feature flag. The current acceptance tests require this to be true